.travis.yml

```diff
@@ -29,18 +29,40 @@ before_install:
 before_script:
   # Install dev. stuff (e.g., selenium drivers).
   - npm run install-dev
+  # Start up processes used in parallel tests.
+  # Note: We're playing a bit fast & loose here bceause selenium standalone needs to "just be ready"
+  # by the time we start our tests.
+  - nohup bash -c "node_modules/.bin/selenium-standalone start 2>&1 &"
 
 addons:
   sauce_connect: true
 
 script:
-  # Run all base checks.
+  # Run all base checks (with `local.phantomjs` browser for functional tests,
+  # and PhantomJS+FF for Karma tests).
   - npm run check-ci
-  - npm run test-func-sauce
-  # Safari runs into VM availability issues on Sauce.
-  # Run as standalone, serially, and grepped to just the advanced build.
-  # https://github.com/walmartlabs/little-loader/issues/5
-  - npm run test-func-sauce-safari
+
+  # Sauce Labs
+  #
+  # Run all different environments in parallel for different `ROWDY_SETTINGS`
+  # values. Flags key:
+  #
+  # - `buffer`: Save test output until _end_ of test run finishes (to make more
+  #   sense of concurrent output).
+  # - `tries=2`: Try each separate test suite 2x before declaring a failures.
+  # - `queue=NUM`: (Not Enabled) Limit concurrency to max of `NUM` tasks at the
+  #   same time.
+  #
+  - >-
+    node_modules/.bin/builder envs test-func-ci --buffer --tries=2
+    '[ { "TEST_FUNC_PORT": 3030, "ROWDY_SETTINGS":"sauceLabs.IE_8_Windows_2008_Desktop" },
+    { "TEST_FUNC_PORT": 3040, "ROWDY_SETTINGS":"sauceLabs.IE_9_Windows_2008_Desktop" },
+    { "TEST_FUNC_PORT": 3050, "ROWDY_SETTINGS":"sauceLabs.IE_10_Windows_2012_Desktop" },
+    { "TEST_FUNC_PORT": 3060, "ROWDY_SETTINGS":"sauceLabs.IE_11_Windows_2012_R2_Desktop" },
+    { "TEST_FUNC_PORT": 3070, "ROWDY_SETTINGS":"sauceLabs.safari_7_OS_X_10_9_Desktop" },
+    { "TEST_FUNC_PORT": 3080, "ROWDY_SETTINGS":"sauceLabs.chrome_latest_Windows_2012_R2_Desktop" },
+    { "TEST_FUNC_PORT": 3090, "ROWDY_SETTINGS":"sauceLabs.firefox_latest_Windows_2012_R2_Desktop" },
+    { "TEST_FUNC_PORT": 3100, "ROWDY_SETTINGS":"local.firefox" } ]'
```

Comment:
And in case it's not clear: This line is replacing

* https://github.com/walmartlabs/little-loader/pull/26/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L82
* https://github.com/walmartlabs/little-loader/pull/26/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L83